firrtl/src/main/scala/firrtl/ir/IR.scala

```diff
@@ -448,14 +449,17 @@ object Formal extends Enumeration {
     clk:  Expression = clk,
     pred: Expression = pred,
     en:   Expression = en,
-    msg:  StringLit = msg
+    msg:  StringLit = msg,
+    args: Seq[Expression] = args
   ): Verification = {
-    Verification(op, info, clk, pred, en, msg, name)
+    Verification(op, info, clk, pred, en, msg, args, name)
   }
 }
 object Verification {
-  def unapply(s: Verification): Some[(Formal.Value, Info, Expression, Expression, Expression, StringLit)] = {
-    Some((s.op, s.info, s.clk, s.pred, s.en, s.msg))
+  def unapply(
+    s: Verification
+  ): Some[(Formal.Value, Info, Expression, Expression, Expression, StringLit, Seq[Expression])] = {
+    Some((s.op, s.info, s.clk, s.pred, s.en, s.msg, s.args))
```

Comment:
@jackkoenig: is this necessary to update for data class?

Reply:
If we keep the data class (implying we're trying to maintain source and binary compatibility), we should not update this (but should instead deprecate it). As is, this is making a breaking change, but I support just breaking it, see my other comment.